lib/ansible/plugins/inventory/docker_machine.py

```diff
@@ -91,8 +91,10 @@ class InventoryModule(BaseInventoryPlugin, Constructable, Cacheable):
 
     NAME = 'docker_machine'
 
+    DOCKER_MACHINE_PATH = get_bin_path('docker-machine')
```

Comment:
This will raise an exception if it can't find `docker-machine`. Better do lazy initialization in `_run_command`, catch the exception there (if it was thrown) and continue as an `AnsibleError`.

Reply:
Done, though it doesn't feel like very Python-like code, I'm sure it can be neater.